rust/src/context.rs

```diff
@@ -104,9 +104,6 @@ impl InstrumentationContext {
             self.stack.push(f);
         }
 
-        if let Err(e) = self.collector.try_send(ev) {
-            error!("error recording memory allocation: {}", e);
-        }
```

Comment:
Hmm yeah, this seems like it might have been leftover from the refactor. Let me test it tomorrow to make sure.